shell/src/main/java/alluxio/cli/fs/command/FreeCommand.java

```diff
@@ -64,6 +68,15 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
       throws AlluxioException, IOException {
     FreeOptions options = FreeOptions.defaults().setRecursive(true).setForced(cl.hasOption("f"));
     mFileSystem.free(path, options);
+    CommonUtils.waitFor("File to be freed.", (v) -> {
+      try {
+        return mFileSystem.getStatus(path).getInMemoryPercentage() == 0;
```

Comment:
in mem percentage or in alluxio percentage?

Reply:
Good point, changed to in Alluxio %